--- application/controller/backend/SettingsController.php	2008-12-09 00:12:28.000000000 +0200
+++ application/controller/backend/SettingsController.php	2008-12-09 00:10:41.000000000 +0200
@@ -20,7 +20,7 @@
	public function index()
	{
		$tree = $this->config->getTree();
+		if (true)
-		if (file_exists($this->getPrivateLabelFile()))
		{
			unset($tree['49-private-label']);
		}

--- application/view/layout/install.tpl	2008-12-09 00:12:28.000000000 +0200
--- application/view/layout/install.tpl	2008-12-09 00:12:28.000000000 +0200
@@ -7,7 +7,7 @@
	<meta http-equiv="content-type" content="text/html;charset=UTF-8" />

	<title>
-		LiveCart Installer
+		LiveCart Community Edition Installer
		{* $PAGE_TITLE *}
	</title>
	<base href="{baseUrl}" />

--- application/view/layout/frontend.tpl	2008-12-09 00:12:28.000000000 +0200
+++ application/view/layout/frontend.tpl	2008-12-09 00:10:41.000000000 +0200
@@ -24,5 +24,5 @@
		{/if}
-	</title>
+	 (Powered by LiveCart)</title>

	<base href="{baseUrl}"></base>

